### DISCUSSION TITLE
Residue-specific fitting 13C MQ CPMG

### DISCUSSION THREAD

**Discussion by OEnny09** [gbouvignies/ChemEx#210] [Q&A] [answered]
Hello!

I am currently trying to use ChemEx to fit 13C MQ CPMG data that was acquired at 600 MHz and I would like to know if it is possible to fit each residue individually instead of using a global fit. If so, how would you do this? Thank you!

**Comment by gbouvignies** [comment, marked as answer]
Hello,

For individual residue fits using a two-state model in ChemEx, utilize the model "2st_rs". Include the option `-d 2st_rs` in your command:

```bash
chemex fit -e [...] -d 2st_rs
```

This configuration allows for residue-specific global exchange parameters.
